impl/src/main/java/com/okta/jwt/impl/jjwt/BaseVerifierBuilderSupport.java

```diff
@@ -66,12 +75,61 @@ public B setConnectionTimeout(Duration connectionTimeout) {
         return self();
     }
 
-    Duration getReadTimeout() {
-        return readTimeout;
+    public String getProxyHost() {
+        return proxyHost;
     }
 
-    public B setReadTimeout(Duration readTimeout) {
-        this.readTimeout = readTimeout;
+    @Override
+    public B setProxyHost(String proxyHost) {
+        this.proxyHost = proxyHost;
+        return self();
+    }
+
+    public int getProxyPort() {
+        return proxyPort;
+    }
+
+    @Override
+    public B setProxyPort(int proxyPort) {
+        this.proxyPort = proxyPort;
+        return self();
+    }
+
+    public String getProxyUsername() {
+        return proxyUsername;
+    }
+
+    @Override
+    public B setProxyUsername(String proxyUsername) {
+        this.proxyUsername = proxyUsername;
+        return self();
+    }
+
+    public String getProxyPassword() {
+        return proxyPassword;
+    }
+
+    @Override
+    public B setProxyPassword(String proxyPassword) {
+        this.proxyPassword = proxyPassword;
+        return self();
+    }
+
+    public int getMaxHttpRetryAttempts() {
```

Comment:
the two retry methods and properties should probably match the SDK names?
`setRetryMaxElapsed`
`setRetryMaxAttempts`

I like the names in this one better, but consistency is probably better overall.

Reply:
Yes, for the sake of consistency lets leave it this way :)

Reply:
Sorry, I meant to rename it like your above comment.

Reply:
Fixed.